docs/docsite/rst/dev_guide/developing_collections_creating.rst

```diff
@@ -81,6 +81,20 @@ To learn more about the ``ansible-galaxy`` command-line tool, see the :ref:`ansi
 
 .. _creating_collection_skeletons:
 
+Creating collections with ansible-creator
+=========================================
+
+`ansible-creator <https://github.com/ansible/ansible-creator/>`_ is designed to quickly scaffold an Ansible collection project.
+
+.. note::
+
+   Installing ansible-creator using the `Ansible Development Tools <https://github.com/ansible/ansible-dev-tools>`_ package is recommended.
```

Comment:
This is passive tense, which we should avoid (even though it was likely written this way elsewhere).

Something like 
"We recommend you install ansible-creator through the 'Ansible Development Tools...'